README.adoc

```diff
@@ -245,40 +245,39 @@ Create the `FaultToleranceTest` class in the
 include::finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java[tags=ft_testing;!javadoc]
 ----
 
-The `@BeforeClass` annotation is placed on a method that executes before any of the test cases. In
+The `@BeforeClass` annotation is placed on a method that executes before any of the other test cases. In
 this case, the `oneTimeSetup()` method retrieves the port number for the Open Liberty server and 
 builds a base URL string that is used throughout the tests.
 
-The `@Before` and `@After` annotations are placed on methods that execute before and after every 
-test case. These methods are generally used to perform any setup and teardown tasks. In this case,
-the setup method creates a JAX-RS client which makes HTTP requests to the `inventory` service. This
-client must also be registered with a JSON-P provider (JsrJsonpProvider) to process JSON resources. 
-The teardown method simply destroys this client instance as well as the HTTP responses, resets the
+The `@Before` and `@After` annotations are placed on methods that execute before and after any 
+other test case. These methods are generally used to perform any setup and teardown tasks. In this case,
+the setup method creates a JAX-RS client, which makes HTTP requests to the `inventory` service. This
+client must also be registered with a `JsrJsonpProvider` JSON-P provider to process JSON resources. 
+The teardown method simply destroys this client instance as well as the HTTP responses and resets the
 changed configuration file and the retry counter.
 
-Let’s break down the test cases:
+Let’s define the test cases:
 
-* `testFallbackForGet()` sends a request to `inventory` service to get the systems properties for a
-hostname, when `system` service is available. Then, it puts the `system` service on maintenance. The
-`inventory` service returns the properties from making a connection to `system` service.
-After that, it makes a second request to `inventory` service to return the system properties for that
-hostname. The fallback method from `inventory` service is called to return the cached properties. 
-Lastly, it asserts if the system properties returned from the first request were more than the second
+* The `testFallbackForGet()` test case sends a request to the `inventory` service to get the systems properties for a
+host name when the `system` service is available. Then, it puts the `system` service on maintenance. The
+`inventory` service returns the properties after making a connection to the `system` service.
+Afterward, it makes a second request to the `inventory` service to return the system properties for that
+host name. The fallback method from the `inventory` service is called to return the cached properties. 
```

Comment:
=> This time the `inventory` service calls the fallback method to return the cached properties.